src/Azure.ApiHub.Sdk/Azure.ApiHub.Sdk.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
```

Comment:
Are you missing the common.props file in your PR?

Reply:
Yes, you are right. The file was missing. I just added it to the repo.